hack/test-cmd.sh

```diff
@@ -682,9 +682,7 @@ __EOF__
 
   ### Perform a rolling update with --image
   # Command
-  kubectl rolling-update frontend --image=kubernetes/pause --update-period=10ns --poll-interval=10ms "${kube_flags[@]}"
-  # Post-condition: current image IS kubernetes/pause
-  kube::test::get_object_assert 'rc frontend' '{{range \$c:=$rc_container_image_field}} {{\$c.image}} {{end}}' ' +kubernetes/pause +'
+  [ ! kubectl rolling-update frontend --image=kubernetes/pause --timeout=1s --update-period=10ns --poll-interval=10ms "${kube_flags[@]}" || true
```

Comment:
Oops. Turns out I forgot to delete those few chars at the beginning of the line before committing, and somehow leaving them in causes a false positive test. But in fixing it so that the test correctly proceeds, things get even more complicated: If we let the update run but fail, we end up with 2 rcs (frontend, and frontend-<uuid>, created by the updater), which messes up subsequent expectations (which assert that only "frontend" should exist). I'd have to somehow identify the auto-named RC the updater generated and delete it. This is starting to get really hairy for a test case that isn't of much value. Any other suggestions?